tests/unit/test_retry.py

```diff
@@ -59,6 +59,16 @@ def test_w_requests_connection_error(self):
         exc = ValueError("testing")
```

Comment:
Side note, I don't understand what this test is doing-- seems to raise a ValueError but the title implies requests.ConnectionError?

Reply:
I saw that, too.  `git blame` says I introduced it in #308.  @cojenco Can you add a commit to this PR renaming that test to `test_miss_w_stdlib_error`?